CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 #### Changes
 * Node: Added SDIFFSTORE command ([#1931](https://github.com/valkey-io/valkey-glide/pull/1931))
+* Node: Added SINTERCARD command ([#1956](https://github.com/valkey-io/valkey-glide/pull/1956))
```

Comment:
Should we follow chronological ordering? Reference: https://keepachangelog.com/en/1.0.0/

Reply:
Good question. I'm not sure if the chronological recommendation in that link refers to just ordering release sections chronologically by date, or if the suggestion is as specific as the ordering of PRs. I don't think its feasible to order PRs by merge date as this is unpredictable. We could order by PR number but I'm not sure if this adds value and the existing CHANGELOG entries are not ordered by PR number. One advantage of not having them in order is that, by avoiding always putting entries at the end of the list, you can avoid regular merge conflicts in this file

Reply:
Oh that's a really good point. Hmm... not sure if it matters then?

Reply:
Yeah personally I don't think it matters